roles/preflight-checks/tasks/check_items.yml

```diff
@@ -69,4 +69,16 @@
 - include: v7k.yml
   when: v7k.enabled
   tags: ['precheck_v7k']
-  run_once: true
\ No newline at end of file
+  run_once: true
+
+- name:  Make sure that if fixed_key is defined then cinder encryption is also enabled
+  assert:
+    that: "cinder.encryption.enabled"
+  when: cinder.encryption.fixed_key is defined
```

Comment:
It's not a good idea to judge by fixed_key is defined or not.
Because it's hard to undefined a variable after we define it, what makes it hard to overwrite.
I prefer to use `fixed_key == None`, of course we should define fixed_key with None value.
```
fixed_key: ~
```